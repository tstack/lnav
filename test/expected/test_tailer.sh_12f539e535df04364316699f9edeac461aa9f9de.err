tailer stderr:
info: load preview request -- 1234
info: exiting...
