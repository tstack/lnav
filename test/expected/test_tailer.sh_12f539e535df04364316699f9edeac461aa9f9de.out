preview of file: {test_dir}/remote-log-dir/*
{test_dir}/remote-log-dir/logfile_access_log.0
{test_dir}/remote-log-dir/logfile_access_log.1

all done!
tailer stderr:
info: load preview request -- 1234
info: exiting...
